src/main.jsx

```diff
@@ -27,6 +27,7 @@ const router = createBrowserRouter([
     path: "/",
     errorElement: <ErrorPage />,
     children: ROUTES,
+      
```

Comment:
Why did you commit this?

Reply:
when i tried to merge it showed me there merge problems

Reply:
That is an empty line